style.css

```diff
@@ -73,6 +79,42 @@ nav li {
   list-style: none;
 }
 
+.visible {
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: auto;
+  position: fixed;
+  z-index: 1;
+  font-size: 20px;
+  background-image: url(/img/image_geometry_menu_1.svg), url(/img/image_geometry_menu_2.svg);
+  background-position: top, right bottom;
+  background-repeat: no-repeat;
+}
+
+.visible a {
+  cursor: pointer;
+}
+
+.visible li {
+  padding-bottom: 5vh;
+}
+
+.closeMenu {
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.toggleNav {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 55px;
+  padding: 0 2vw;
+}
```

Comment:
- [ ] Please keep in mind that it's always better to follow a given design, it's a chance to practice on it. So, please take a look at the two images from the [original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) below and make the marked elements' styles to your project.
![Screenshot](https://user-images.githubusercontent.com/47625407/123839715-bc300d00-d90d-11eb-955d-f082ff873e71.png)
![Screenshot1](https://user-images.githubusercontent.com/47625407/123839734-c0f4c100-d90d-11eb-8f84-916d18915f38.png)